src/scenes/home/mentorRequest/mentorRequest.js

```diff
@@ -55,7 +55,7 @@ export default class MentorRequest extends Component {
     this.state.services.map(service => ({ value: service.id, label: service.name }))
 
   buildMentorOptions = () =>
-    this.state.mentors.map(mentor => ({ value: mentor.id, label: mentor.email }))
+    this.state.mentors.map(mentor => ({ value: mentor.id, label: `${mentor.last_name} ${mentor.first_name}` }))
 
```

Comment:
Comma separating the mentors' last name and first name could make this a better UX for a user.  Since we're sorting the list alphabetically by last name.  Right now they're just separated by space, and it may not be apparent if it's the first or last name that comes first.

Example: 'Hampton, John'

Reply:
@jjhampton done.